packages/web-app-files/src/components/SideBar/Details/FileDetailsMultiple.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div id="oc-file-details-sidebar">
+    <div class="files-preview">
+      <div class="files-preview__body">
+        <oc-icon class="preview-icon" size="xxlarge" variation="passive" name="file_copy" />
+        <span class="preview-text" data-testid="selectedFilesText" v-text="selectedFilesString" />
+      </div>
+    </div>
+    <div>
+      <table class="details-table" aria-label="detailsTableLabel">
```

Comment:
aria-label doesn't work ;-) you need to add a computed property `detailsTableLabel` and make the prop dynamic (`:aria-label`)